packages/api-kit/.npmignore

```diff
@@ -0,0 +1,4 @@
+node_modules
+.idea
+.vscode
+dist
```

Comment:
You should remove dist from npmignore, because the dist folder should be published to the npm package.

Reply:
Makes sense,
when you wrote me to add it there I wasn't sure if I was missing anything